scripts/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# build stage
+FROM archlinux/base AS builder
+
+# install build dependencies
+RUN pacman --noconfirm -Sy \
+    gcc \
+    git \
+    rust \
+    diffutils \
+    file \
+    awk \
+    make
+
+# build
```

Comment:
You still need to copy the source code into docker container. See error: https://cloud.docker.com/u/toxrust/repository/registry-1.docker.io/toxrust/tox-node/builds/c2bf421a-0c59-4644-bfcd-226dc7c80a34

Fix:

```
# copy local files to container
ADD . /tox-node
WORKDIR /tox-node

# build
RUN cargo build --release
```

But you'd better put Dockerfile into root instead of scripts/

Reply:
done